api/cont.js

```diff
@@ -114,7 +114,17 @@
                 continuation
                     .extend({source: sourceFragment}, idx, sourceLength, value)
                     .evalReference();
-            });
+            };
+
+            if (typeof source.length === 'number') {
+                $.each(source, eachCallback);
+            }
+            else {
```

Comment:
I am not sure why Zepto's `$.each` fails to iterate but we did proved that it failed to iterate. All I did here was pulling out Zepto's `$.each` logic so that we are not using what is assign to `$.each`. This is just one of the code variation that I tried for in attempt to target the fault. We'll do proper cleanup once I verified this does fix the issue.

Referring to the screenshot I provided, you can see in the logging that `$.each` function worked in the previous template object iteration but failed on the second template object iteration. "When" `$.each` fails to iterate is random. This means that it is not failing at specific template object nor at specific position.

So we know for certain the following:
* `$.each` is assigned properly before the iteration
* We know the `$.each` callback never got executed
* We know that pulling out the exact logic of `$.each` doesn't fall thru to desktop (Waiting for a live test to verify this)

So how did the assignment of `$.each` randomly disappear? I really have no idea how I can dig even deeper than this.